packages/react-adminer/src/components/Edit.tsx

```diff
@@ -172,6 +172,21 @@ export const Edit: React.FC<Props> = ({ entityConfig, entityName, id }) => {
 	const isErrorNullable = Object.keys(errorNullable).length > 0;
 	const hasChanges = Object.keys(getPayload()).length >= (targetPrimitiveFields.length === 0 ? 0 : 1);
 	const Link = router?.components.Link;
+	interface DateFormatsMap {
```

Comment:
Pamatuj na to, že jsi v těle reactu, tohle se vola při každém průchodu (renderu).  Ideální to je dát mimo ten render.

Zároveň to používáš pro jednu věc, takže bych to nezakládal a rovnou to napral inline k tomu objektu:
```
const DATE_FORMATS: [key: string]: string | undefined= {
	datetime: 'YYYY-MM-DD hh:mm:ss',
	date: 'YYYY-MM-DD',
};
```
Ještě bych zvolil jiný zápis:
```
const DATE_FORMATS: Record<string, string|undefined>= {
	datetime: 'YYYY-MM-DD hh:mm:ss',
	date: 'YYYY-MM-DD',
};
```

Poznámka - typescript je převeden na JS a odstraňuje interface, typy, atd...